### ISSUE TITLE
TypeError: can't access dead object (at "ScriptEditor")

### ISSUE THREAD

**Issue by pstelzer** [danielyxie/bitburner#2950]
### How did this happen?

Please fill this information with details if relevant.

- [x] Save file
- [x] Steps to reproduce

### Environment

* Error: TypeError: can't access dead object
* Page: ScriptEditor
* Version: v1.4.0 (a3c599e6)
* Environment: Production
* Platform: Browser
* UserAgent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:96.0) Gecko/20100101 Firefox/96.0
* Features: lang=de cookiesEnabled=true doNotTrack=1 indexedDb=true
* Source: https://danielyxie.github.io/bitburner/dist/ext/monaco-editor/min/vs/editor/editor.main.js

### Save

[bitburnerSave_1644584522_BN1x0.zip](https://github.com/danielyxie/bitburner/files/8048815/bitburnerSave_1644584522_BN1x0.zip)

### Reproduction Step

- Open ScriptEditor

**Comment by pstelzer** [comment]
This issue is still occuring even after an 'Install Augmentations' soft reset. Every time I try to open the Script Editor I end up in Recovery Mode with the error message above.
I am unable to edit or create new scripts and I'm not exactly sure what I did that lead to this.
One possible explanation might be that I deleted a script via the terminal and this script could still have been open in the editor, but I can't verify that right now. I don't know what else the 'dead object' might refer to.

The attached save is the latest after installing new augmentations.

[bitburnerSave_1645176421_BN1x0.json.txt](https://github.com/danielyxie/bitburner/files/8095935/bitburnerSave_1645176421_BN1x0.json.txt)
